Submitty/Submitty
# [Bugfix:TAGrading] Fix Progress Bar for Peer Graders

### What is the current behavior?
When logged in as a peer grader, progress bar displays 100% even when one has not graded a single component.

### What is the new behavior?
The progress bar now accurately displays the number of components you have left to grade.

### To Test
1. As the instructor, create a gradeable with peer grading components.
2. Login as at least three students and submit to this gradeable.
3. As the instructor, in the gradeable's configuration, generate peer grading assignments in the peer grading tab.
4. As one of the students who submitted, grade one component of another student. Refresh the page and notice the progress bar updates.

Solves issue #10050 .
